modules/katamari/src/test/ts/atomic/api/data/ResultValueTest.ts

```diff
@@ -61,7 +61,7 @@ UnitTest.test('Checking value.getOrDie() does not throw', () => {
 });
 
 UnitTest.test('Checking value.or(oValue) === value', () => {
-  fc.assert(fc.property(fc.integer(), fc.integer(), (a, b) => {
+  fc.assert(fc.property(fc.integer(), fc.integer(), (a, _b) => {
```

Comment:
Seems very odd that we generate an integer  here and don't use it.

Reply:
I removed it.

Reply:
I'm wondering if `b` should have actually been used here? As otherwise this test isn't really doing anything as it always returns `Result.value(a)` so I'm wondering if `Result.value(b)` should have been used in the or?

Reply:
In saying that though, that's still not really testing the or is working. So maybe this should have two assertions like so?
```
Assert.eq('eq', Result.value(a), Result.value(a).or(Result.value(b)), tResult());
Assert.eq('eq', Result.value(b), Result.error(a).or(Result.value(b)), tResult());
```

Reply:
Yeah that makes much more sense. I'll update it to your version.

Reply:
Done